config/rtlwrapper.js

```diff
@@ -0,0 +1,15 @@
+import { BrowserRouter } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+export const mountWithRouter = (Component) => {
+    return render(<BrowserRouter>{Component}</BrowserRouter>);
+};
+
+export const mountWithRouterAndProvider = (Component, store) => {
+    return render(
+        <Provider store={store}>
+            <BrowserRouter>{Component}</BrowserRouter>
```

Comment:
Wondering if MemoryRouter would be better to use?

Reply:
I tried it and it didn't work :(